devops/azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+# This pipeline relies on the analysis.yml template located in the
+# repository resource descibed below
+resources:
+  repositories:
+  - repository: templates
+    type: github
+    endpoint: Team1-Project3
+    name: 1053-August-Duet-Project-Registry/project-registry-devops
+    ref: refs/heads/feature-templates
+
+# This pipeline will trigger on commits or pull-requests made to the branch-dev
+# or main branches.  Note that static analysis with SonarCloud is only performed
+# on "long-lived" branches, which are designated as 'main' and any branch name
+# prefixed with 'branch-'.
+trigger:
+- branch-dev
+- main
+pr:
+- branch-dev
+- main
+
+pool:
+  vmImage: ubuntu-latest
+
+# These variable groups are required for sonarCloud and Discord secrets to be
+# available in the analysis template
+variables:
+- group: project-registry-sonarcloud
+- group: project-registry-discord
+
+stages:
+- stage: build
+  jobs:
+  - job: analyze
+
+    steps:
+    - checkout: self
+    - checkout: templates
+    - template: ./templates/analysis.yml@templates
+      parameters:
+        microservice: account-microservice
+        workDir: ./project-registry-account-microservice/
```

Comment:
you could probably use the Build.Repository.Name variable here. (we like every opportunity to reduce the number of lines in files that this that you'd need to change to apply it to a new project/repo. = more maintainability.)

Reply:
The output of Build.Repository.Name is '1053-August-Duet-Project-Registry/project-registry-account-microservice', so it does not match the needs of this variable.